movie_trainer.py

```diff
@@ -0,0 +1,60 @@
+import os
+from dotenv import load_dotenv
+import pandas as pd
+from pymongo import MongoClient
+from sklearn.feature_extraction.text import CountVectorizer
+from sklearn.metrics.pairwise import cosine_similarity
+
+load_dotenv()
+
+class MovieTrainer:
+    @staticmethod
+    def load_movies_from_db():
+        mongo_uri = os.getenv("MONGO_URI")
+        client = MongoClient(mongo_uri)
+        db = client["movieDB"]
+        collection = db["movieDetails"]
+
+        projection = {
+            "_id": 1,
+            "title": 1,
+            "genres": 1,
+            "cast": 1,
+            "crew": 1,
+            "keywords": 1
+        }
+
+        movies = list(collection.find({}, projection))
+        return pd.DataFrame(movies)
+
+    @staticmethod
+    def create_soup(movie):
```

Comment:
same with static, you're even calling it using self, so there's no need for it to be static
also, rename the parameter to `movies_df` for consistency